frontend/src/pages/Volunteers.tsx

```diff
@@ -353,6 +354,30 @@ const Volunteers = () => {
     setSearchWord(undefined);
   };
 
+  const onExport = async () => {
+    const { data: volunteersData } = await getVolunteersForDownload(
+      volunteerStatus,
```

Comment:
astea sunt aceleasi filtre ca la getMany normal. Oare sa facem o metoda sa ni le intoarca? Ca sa nu modificam in 2 parti daca se modifica?

Reply:
O sa las un TODO.

Reply:
Nu mai e cazul, am inteles de la @birloiflorian ca trebuie sa le trimitem separat in useQuery ca sa faca trigger la changes